tribute.html

```diff
@@ -15,7 +15,6 @@ <h1 id="title">Cat familly</h1>
         <figure id="img-div">
             <img id="image" src="https://images.pexels.com/photos/20787/pexels-photo.jpg?auto=compress&cs=tinysrgb&w=800">
             
-
             <figcaption id="img-caption">Mr BossetCat The Asian wildcat (Felis silvestris ornata) ranges east of the Caspian Sea into China.</figcaption>

```

Comment:
- [ ] I think it is better to phrase your request in a suggestion, e.g "I kindly suggest you use 2 spaces..." because they help make it easy to know what is it you want changed while being polite 😉

Reply:
- [ ] Great work, Your comment is polite, clear, & offers the reason for the change 👏🏾